docs/how-to-guides/demos/isis-demos.md

```diff
@@ -0,0 +1,36 @@
+# ISIS Demos
+
+<script src="https://asc-public-docs.s3.us-west-2.amazonaws.com/common/scripts/isis-demos/jquery-3.7.1.min.js"></script>
+<link href="../../../css/isis-demos.css" media="all" rel="stylesheet"/>
+
+
+<script type="text/javascript">
+if (typeof window.isisDemosLoaded == 'undefined') {
+    var isisDemosLoaded = true;
+    $.getScript("https://asc-public-docs.s3.us-west-2.amazonaws.com/common/scripts/isis-demos/easeljs-0.8.1.min.js").done( function(s,t) { $.getScript("../../../js/isisDemos.js");});
+}
+</script>
+
+## Image Data Size Calculator
+<div class="app-container" id="isis-image-data-size"></div>
+
+## Cube Demo
+<div class="app-container" id="isis-cube"></div>
+
+## Pixel Demo
+<div class="app-container" id="isis-pixels"></div>
+
+## Multiplier Demo
+<div class="app-container" id="isis-multiplier"></div>
+
+## Subpixels Demo
+<div class="app-container" id="isis-subpixels"></div>
+
+## Special Pixels Demo
+<div class="app-container" id="isis-special-pixels"></div>
```

Comment:
I think these need to go into https://astrogeology.usgs.gov/docs/concepts/isis-fundamentals/cube-format/

Reply:
I placed `cube demo`, `multiplier demo`, and `subpixel demo` on the cube-format page, and the `Image data size calculator` on the bit-type page.